plugins/PrivacyManager/API.php

```diff
@@ -400,6 +409,36 @@ public function executeDataPurge(
         }
     }
 
+    /**
+     * @internal
+     */
+    public function getComplianceStatus(string $idSite, string $complianceType): array
+    {
+        if ($this->featureFlagManager->isFeatureActive(PrivacyCompliance::class) === false) {
```

Comment:
`isFeatureActive` returns bool, so maybe we don't need to strictly check for false? Makes you read the whole line to see what it checks for.

```suggestion
        if (!$this->featureFlagManager->isFeatureActive(PrivacyCompliance::class)) {
```

Reply:
I've put the false at the start, I find `!` is always a bit hard to see sometimes. Just being extra verbose for readability.